internal/querycoordv2/meta/resource_manager.go

```diff
@@ -258,15 +258,18 @@ func (rm *ResourceManager) TransferNode(sourceRGName string, targetRGName string
 	if sourceCfg.Requests.NodeNum < 0 {
 		sourceCfg.Requests.NodeNum = 0
 	}
+	// Special case for compatibility with old version.
+	if sourceRGName != DefaultResourceGroupName {
```

Comment:
keep compatible with old version sdk.
Once using `TransferFrom` and not change `Limits` with `Requests`.
DropResourceGroup will be rejected forever.